frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import PagedJobsTable from "main/components/Jobs/PagedJobsTable";
 import Accordion from "react-bootstrap/Accordion";
-import TestJobForm from "main/components/Jobs/TestJobForm";
+// import TestJobForm from "main/components/Jobs/TestJobForm";
```

Comment:
Ack!  What have we said about commenting out code in production?

It's fine to comment out while testing, but when you make a PR, the commented out code should go bye-bye.